python/lib/dependabot/python/language.rb

```diff
@@ -27,7 +27,7 @@ class Language < Dependabot::Ecosystem::VersionManager
       PRE_INSTALLED_PYTHON_VERSIONS = T.let(
         PRE_INSTALLED_PYTHON_VERSIONS_RAW.map do |v|
           Version.new(v)
-        end.sort,
```

Comment:
Can we document inline why reverse is needed?

Also, we should add a spec as @thavaahariharangit pointed out